client/src/components/AuditAdmin/Setup/Review/Review.tsx

```diff
@@ -137,14 +145,41 @@ const Review: React.FC<IProps> = ({
       auditSettingsQuery.data
     )
 
+  const isContestNameStandardizationNeeded =
+    !!contestNameStandardizations &&
+    Object.values(contestNameStandardizations.standardizations).length > 0
+  const isContestNameStandardizationOutstanding =
+    !!contestNameStandardizations &&
+    Object.values(
+      contestNameStandardizations.standardizations
+    ).some(jurisdictionStandardizations =>
+      Object.values(jurisdictionStandardizations).some(
+        cvrContestName => cvrContestName === null
+      )
+    )
+  const isContestNameStandardizationComplete =
+    !!contestNameStandardizations &&
+    !(
+      isContestNameStandardizationNeeded &&
+      isContestNameStandardizationOutstanding
+    )
```

Comment:
This block above is all existing code, just moved down and with variable names renamed (standardizations ➡️ contestNameStandardizations)